docs/tuning_fledge.rst

```diff
@@ -762,6 +767,113 @@ When collection is enabled the following counters will be collected for the stor
+
+.. note::
+
+   The audit log is also used by the *FogLAMP Manage* product to determine if changes have been made locally to the instance. Therefore the retention period for audit log data must be greater that the frequency with which that product is collecting this data from the instance.
+
+The task log is used internally within Fledge to track the state of running tasks as well as to give the history of tasks that have run for support purposes, this data is included in the support bundles. Therefore the retention should be such that there is sufficient history to cover any period that might be needed to diagnose issues within Fledge. Also the period should not be so small that it risks the data for a running task being purged before the task has completed. As a guideline it must never be less than 1 day, it is recommended to keep at least 7 days to allow for some history to be available for diagnostic purposes.
+
+As well as the configuration of the retention period for the various logs the other tuning that can be done is the frequency of the execution of the system purge process. This is done in the *Schedules* menu item and is the tasks named *purge_system*. The default is to run it every 23 hours and 50 minutes.
+
+.. note::
+
+   If you run the system purge every 24 hours and, you retain 7 days worth of data for the statistics, you will have 8 days of data stored at the peak of storage use. This is because when the process runs it will reduce the data down to 7 days, but as soon as it has completed new data will accumulate until it is next run a day later. The same is obviously also true for the task and audit data.
+
+Purge Process
+-------------
+
+The purge process is probably the more important process to tune of the two, it manages the storage for the reading data that is the more dynamic and larger data set of the two controlled by purge processes. As with the purge system process above the configuration of how the purge process runs is available in the *Configuration* menu item in the category *Utilities::Purge*.
+
++---------------+
+| |PurgeConfig| |
++---------------+
+
+The details of each of the options are covered elsewhere in the documentation, but the salient points will be repeated here. The operation of the purge process reduces the number of readings that are retained in the readings storage subsystem using two parameters;
+
+   - the age of the reading
+
+   - the number of readings
+
+The age is set in hours and any reading older than this age is a candidate to be removed from the readings data. The purge process also looks at the number of readings stored and will remove the oldest, even if they are newer than the age to be retained if the number exceeds the maximum rows to retain value.
```

Comment:
- The age is set in hours and any reading older than this age -> The age is set in hours. Any reading older than this age
- "...if the number exceeds the maximum rows to retain value." The name of the configuration parameter ("maximum rows to retain") needs to be offset somehow. This reads as an awkward sentence without it. You have been using _italics_ for other commands and parameters. I checked the Fledge GUI; the name of the parameter is "Max rows of data to retain."